.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
+# #!/usr/bin/env sh
+# . "$(dirname -- "$0")/_/husky.sh"
 
-npm test
+# npm test
```

Comment:
You definitely shouldn't comment the pre-commit hook out!
If you've had to do this then you've probably got tests that don't pass - we'd need to fix the tests/server instead of just not running them